app/controllers/miq_request_controller.rb

```diff
@@ -415,16 +415,26 @@ def get_request_tab_type
     end
   end
 
+  def remote_and_global_requestors(requestor)
```

Comment:
yes, suggested is the right way to fix reported bug (especially with assumption that new methods in `User` and `Condition` would be used in other places too). 
Does it make sense to keep PR as is for `fine` branch (in case there is time constrain and this BZ need to be fixed now) and start working on master branch to do it in the right way.
@gtanzillo what do you think ?